www/doc/1.7/installing.html

```diff
@@ -75,43 +83,44 @@ <h2>Installing Herbie from source</h2>
   <p>
     This command installs Herbie and its dependencies, compiles it for
     faster startup, and places the <code>herbie</code> executable in
-    your Racket user path (example paths for Racket 8.5):
+    your Racket user path (example paths for Racket 8.9):
   </p>
 
   <ul>
-    <li>On Windows, <code>AppData\Roaming\Racket\8.5\bin</code> in your user folder.</li>
-    <li>On macOS, <code>Library/Racket/8.5/bin</code> in your home folder.</li>
-    <li>On Linux, <code>.local/share/racket/8.5/bin</code> in your home directory.</li>
+    <li>On Windows, <code>AppData\Roaming\Racket\8.9\bin</code> in your user folder.</li>
+    <li>On macOS, <code>Library/Racket/8.9/bin</code> in your home folder.</li>
+    <li>On Linux, <code>.local/share/racket/8.9/bin</code> in your home directory.</li>
   </ul>
 
   <p>
-    You can run <code>herbie</code> from that directory, or add it to
-    your executable path. Once Herbie is installed and working
+    Please note that the path of the binary will be different on Linux if you 
+    have opted to use Snap. You can run <code>herbie</code> from that directory,
+    or add it to your executable path. Once Herbie is installed and working
     correctly, check out the <a href="tutorial.html">tutorial</a>.
   </p>
 
   <h2>Installing Herbie from a package</h2>
 
   <p>
-    This installation method supports Windows, macOS, and Linux for x86-64.
-    This method of installation will fail for Apple M1 systems and other ARM architectures.
-    Once Racket is installed, install Herbie from a package with:
+    This installation method supports Windows ((note that you will need some way to run make),
```

Comment:
I do not think you need make to install Herbie from the package manager.

Reply:
When I first tried doing it this way, the raco pkg install command failed with an error on make not being detected - perhaps I did something wrong when I was first trying this?